sled-agent/Cargo.toml

```diff
@@ -56,7 +56,7 @@ tokio-util = { version = "0.7", features = ["codec"] }
 toml = "0.5.9"
 uuid = { version = "1.2.1", features = [ "serde", "v4" ] }
 vsss-rs = { version = "2.0.0", default-features = false, features = ["std"] }
-zone = "0.1"
+zone = { version = "0.2", default-features = false, features = ["async"] }
```

Comment:
The default feature is `sync`, and since we're not using that feature, I *only* want the `async` APIs.